addons/mrp/__manifest__.py

```diff
@@ -16,6 +16,7 @@
         'data/digest_data.xml',
         'data/mail_templates.xml',
         'data/mrp_data.xml',
+        'data/mail_message_subtype_data.xml',
```

Comment:
useless not used

Reply:
Hello @amoyaux,

As per the spec I've added the new sub stage in data which is define in manifest. Am i missing something? I didn't get you by useless.

https://github.com/odoo/odoo/pull/163289/files#diff-33b554a878f9398085343251201a13709e08b4b291eca3b0f7e8a2f3ff2ba145R1-R46

Thanks.

Reply:
Read at _track_subtype method...